### PR TITLE
State processing value fixes

### PR DESCRIPTION
**Pull request by uhurusurfa**
Fix missing DIRECTION and FILE_NAME fields when logging message state to the database

### CODE REVIEW

File: Server/src/main/java/org/openas2/processor/receiver/AS2MDNReceiverHandler.java [hunk 1]
```diff
@@ -49,8 +49,9 @@ public void handle(NetModule owner, Socket s)
 
 		AS2Message msg = new AS2Message();
 
-        Map<Object, Object> options = new HashMap<Object, Object>();
-		options.put("DIRECTION", "SEND");
+
+		msg.getOptions().put("DIRECTION", "SEND");
```
Comment (VboxNick) [inline Server/src/main/java/org/openas2/processor/receiver/AS2MDNReceiverHandler.java]: For this purpose it's better to use org.openas2.message.BaseMessage#setOption

File: Server/src/main/java/org/openas2/processor/receiver/AS2ReceiverHandler.java [hunk 2]
```diff
@@ -70,8 +70,9 @@ public void handle(NetModule owner, Socket s) {
 
         byte[] data = null;
         BufferedOutputStream out;
-        Map<Object, Object> options = new HashMap<Object, Object>();
-		options.put("DIRECTION", "RECEIVE");
+
+		msg.getOptions().put("DIRECTION", "RECEIVE");
```
Comment (VboxNick) [inline Server/src/main/java/org/openas2/processor/receiver/AS2ReceiverHandler.java]: For this purpose it's better to use org.openas2.message.BaseMessage#setOption

**Comment by uhurusurfa** [comment]
@VboxNick  - good catch - have changed ... and I will do the squash next time

**Comment by VboxNick** [comment]
@uhurusurfa Great, thanks. I did squash via GitHub web interface.

**Pull request merged**